projects/systelab-components/tsconfig.lib.json

```diff
@@ -5,6 +5,7 @@
         "target": "es2015",
         "declaration": true,
         "inlineSources": true,
+        "noImplicitOverride": true,
```

Comment:
Nice change! I was missing this sometimes adding override as a comment // Override :D

Reply:
Yes, it's nice, it's new from typescript 4.3